### PR TITLE
<Page/> - migrate to Stylable

### PR DESCRIPTION
**Pull request by daugerdas**
### 🔦 Summary

Migrate `<Page/>` to Stylable

Description: [DSM-1052](https://jira.wixpress.com/browse/DSM-1052)

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6241

File: src/Page/Page.js [hunk 1]
```diff
@@ -407,7 +412,7 @@ class Page extends React.PureComponent {
       return (
         <div
           data-hook="page-gradient-class-name"
-          className={`${s.gradientBackground} ${this.props.gradientClassName}`}
+          className={`${classes.gradientBackground} ${this.props.gradientClassName}`}
```
Comment (mykas) [inline src/Page/Page.js:415]: Lets use here `st` command to merge classnames.

File: src/Page/Page.st.css [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+}
+
+.contentHorizontalLayout:contentFullWidth {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.contentHorizontalLayout:horizontalScroll {
+  overflow-x: auto;
+
```
Comment (mykas) [inline src/Page/Page.st.css:199]: lets remove empty spaces

File: src/Page/Page.st.css [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+  padding-top: 30px;
+  padding-bottom: value(headerBottomPadding);
+}
+
+.pageHeaderContainer .headerWrapper:minimized {
+  box-sizing: content;
+  min-height: value(minimizedHeaderWrapperMinHeight);
+  padding-top: value(minimizedHeaderWrapperTopPadding);
+  padding-bottom: value(minimizedHeaderWrapperBottomPadding);
+
```
Comment (mykas) [inline src/Page/Page.st.css:116]: lets remove this space

**Review by mykas — CHANGES_REQUESTED** [review]

File: src/Page/Page.st.css [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+  overflow-y: auto;
+  width: 100%;
+  position: relative;
+  height: 100%;
+}
+
+.pageHeaderContainer {
+  -st-states: minimized, hasTail;
+  position: relative;
+
```
Comment (mykas) [inline src/Page/Page.st.css:96]: lets remove the space

File: src/Page/Page.st.css [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+  -st-from: "../Foundation/stylable/mixins/calc_rgba.js";
+  -st-default: calc_rgba;
+}
+
+:vars {
+  page-side-padding: 48px;
+  headerBottomPadding: 24px;
+  image-container-offset: 30px;
+  pagebottompadding: 48px;
+  /* Should be synced with JS Page value */
```
Comment (mykas) [inline src/Page/Page.st.css:26]: Maybe could be achieved through this: https://stylable.io/docs/guides/runtime

**Review by mykas — APPROVED** [review]
Good job! 🏆  Let's test this before meging as we talked about in projects that use this component.

**Pull request merged**